static/templates/message_body.hbs

```diff
@@ -1,38 +1,25 @@
-<div class="message_top_line">
-    {{#unless status_message}}
-    <span class="message_sender sender_info_hover no-select">
-        {{#if include_sender}}
-        <span title="{{t 'View user card' }} (u)">
-            {{> message_avatar ~}}
-            <span class="sender_name auto-select" role="button" tabindex="0">{{msg/sender_full_name}}{{> status_emoji msg/status_emoji_info}}</span>
-            {{#if sender_is_bot}}
-            <i class="zulip-icon zulip-icon-bot" aria-label="{{t 'Bot' }}"></i>
-            {{/if}}
-        </span>
+{{#unless status_message}}
+<span class="message_sender sender_info_hover no-select">
+    {{#if include_sender}}
+    <span title="{{t 'View user card' }} (u)">
+        {{> message_avatar ~}}
+        <span class="sender_name auto-select" role="button" tabindex="0">{{msg/sender_full_name}}{{> status_emoji msg/status_emoji_info}}</span>
+        {{#if sender_is_bot}}
+        <i class="zulip-icon zulip-icon-bot" aria-label="{{t 'Bot' }}"></i>
+        {{/if}}
+        {{#if edited_alongside_sender}}
+        {{> edited_notice}}
         {{/if}}
     </span>
-    {{/unless}}
-
-    {{#if status_message}}
-    {{> me_message}}
-    {{/if}}
-
-    <span class="alert-msg pull-right"></span>
-
-    <a href="{{ msg/url }}" class="message_time{{#if msg.locally_echoed}} notvisible{{/if}}{{#if status_message}} status-time{{/if}}">
-        {{#unless include_sender}}
-        <span class="copy-paste-text">&nbsp;</span>
-        {{/unless}}
-        {{timestr}}
-    </a>
-
-    {{#if edited_alongside_sender}}
-    {{> edited_notice}}
     {{/if}}
+</span>
+{{/unless}}
 
-    {{> message_controls}}
+{{#if status_message}}
+{{> me_message}}
+{{/if}}
 
-</div>
+<span class="alert-msg"></span>
```

Comment:
This lost the `pull-right` class; can you do a screenshot showing a message that contains one of these `alert-msg` elements? I've forgotten what they are.

Reply:
<img width="1721" alt="image" src="https://user-images.githubusercontent.com/25124304/206638760-c386d495-3fdb-4cc4-a7ca-214ccf36bea7.png">